packages/rolldown/tests/fixtures/plugin/write-bundle/_config.ts

```diff
@@ -25,8 +27,9 @@ export default defineTest({
           expect(chunk.facadeModuleId).toBe(entry)
           expect(chunk.exports.length).toBe(0)
           expect(chunk.imports).length(0)
-          expect(chunk.moduleIds).toStrictEqual([entry])
-          expect(Object.keys(chunk.modules).length).toBe(1)
+          // The `foo.js` should be include `modules/moduleIds` even it is empty.
```

Comment:
What's the different between empty modules and modules that got tree-shaked? Do tree-shaked modules need to included too?

Reply:
The chunk render result is same, only the API need to including empty module, the tree-shaked modules  don't need to included .